Userland/Libraries/LibGfx/PathPainter.h

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2022, Daniel Oberhoff <daniel@danieloberhoff.de>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <LibGfx/FillPainter.h>
+#include <LibGfx/Rasterizer.h>
+#include <LibGfx/StrokePainter.h>
+
+namespace Gfx {
+class PathPainter {
+public:
+    enum class StrokeKind { OpenStroke,
+        ClosedStroke,
+        NoStroke };
+    enum class FillKind { Filled,
+        NotFilled };
```

Comment:
Lets have all enumerators on the same column

```suggestion
    enum class StrokeKind {
        OpenStroke,
        ClosedStroke,
        NoStroke
    };

    enum class FillKind {
        Filled,
        NotFilled
    };
```

Reply:
> Lets have all enumerators on the same column
> 
> 
> 
> ```suggestion
> 
>     enum class StrokeKind {
> 
>         OpenStroke,
> 
>         ClosedStroke,
> 
>         NoStroke
> 
>     };
> 
> 
> 
>     enum class FillKind {
> 
>         Filled,
> 
>         NotFilled
> 
>     };
> 
> ```

I actually think clang format did that, will do!

Reply:
done